src/services/query.ts

```diff
@@ -106,3 +108,17 @@ export class QueryTermsService {
     });
   }
 }
+
+const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+  const prefLabelA = a.prefLabels[0]?.value;
+  const prefLabelB = b.prefLabels[0]?.value;
+  if (prefLabelA < prefLabelB) {
+    return -1;
+  }
+
+  if (prefLabelA > prefLabelB) {
+    return 1;
+  }
+
+  return 0;
```

Comment:
Or we can use `return prefLabelA.localeCompare(prefLabelB)` - a bit more concise?

Reply:
Nice! Changed.